### ISSUE TITLE
CX_SY_CONVERSION_NO_NUMBER Error in Method GET_VALUE_TYPE

### ISSUE THREAD

**Issue by ghost** [abap2xlsx/abap2xlsx#385]
Hello community,

in the method GET_VALUE_TYPE, of the class ZCL_EXCEL_WORKSHEET, I get the error CX_SY_CONVERSION_NO_NUMBER if I use a field with a conversion exit function module CONVERSION_EXIT_TSTAM_OUTPUT. The FM converts a date field from packed decimal to character format. The export parameter ep_value of the method has always the format P(8) in the context of this conversion. To soften the effects of the error, I catch it on the following way:

```
      IF sy-subrc <> 0.
        " MESSAGE ID SY-MSGID TYPE SY-MSGTY NUMBER SY-MSGNO
        "         WITH SY-MSGV1 SY-MSGV2 SY-MSGV3 SY-MSGV4.
      ELSE.
        Try.
          ep_value = l_value.
        Catch CX_SY_CONVERSION_NO_NUMBER.

        EndTry.
      ENDIF.
```

I read also about the same problem here http://help.sap.com/erp2005_ehp_04/helpdata/en/79/c554a6b3dc11d5993800508b6b8b11/content.htm

But in the case of an exception the ep_value is empty. Is it possible to solve this problem on another way?

Thanks for hints.

Best regards
Stefan

**Comment by StefanSchmoecker** [comment]
Closed --> See issue #357

**Comment by ghost** [comment]
Hello Stefan,
thank you very much.
Best regards
Stefan
